iwyu_output.cc

```diff
@@ -1426,6 +1426,59 @@ void CalculateIwyuForFullUse(OneUse* use,
   }
 }
 
+// This function removes all OneIncludeOrForwardDeclareLine's a that are desired
+// but not present but transitively included by some
+// OneIncludedOrForwardDeclareLine b that is desired and present
+// and adds a's symbol counts to b.
+vector<OneIncludeOrForwardDeclareLine> TolerateTransitively(vector<OneIncludeOrForwardDeclareLine> const & oiofdls,
```

Comment:
Can you call variables of type `OneIncludeOrForwardDeclareLine` `line[s]` instead? I like the `oiofdls` acronym, but it's hard to read and spell correctly :)

Reply:
`TolerateTransitively` does not describe what this method does, more a general attitude :)

I wonder what a good name would be... `RemoveTransitivelyProvided` or something? Shorter, preferably.